container/pureos/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM subversiveowl/pureos8-base
+MAINTAINER subversive-owl "subversive.owl@yandex.com"
+
+COPY escalate.sh /tmp/escalate.sh
+COPY pkg.sh /tmp/pkg.sh
+
+RUN apt-get update -y
+RUN chmod +x /tmp/pkg.sh && \
+	/tmp/pkg.sh && \
+	apt-get clean
+
```

Comment:
Does this require a `-y`? I did not test the `ubuntu` containers yet..

Reply:
it does if there are multiple packages, yes